src/app/(not-authenticated)/ajout-structure/page.tsx

```diff
@@ -0,0 +1,15 @@
+import { Metadata } from "next";
+import { ReactElement } from "react";
+
+export default function AjoutStructure(): ReactElement {
+  return (
+    <div>
+      <h1>Ajout d&apos;une structure</h1>
```

Comment:
```suggestion
      <h1>Ajout d&apos;une structure</h1>
```
Pour faire des apostrophes, j'utilise le raccourci de clavier CTRL+MAJ+U2019 ’ qui permet de faire une apostrophe lisible aussi dans le code. Qu'est-ce que tu en penses ?

Reply:
Yes, mieux :)